deployments/bsctestnet/GovernorBravoDelegate.json

```diff
@@ -0,0 +1,1153 @@
+{
+  "address": "0x0A79EEFA81098B8BD3c8B11Fc0113f1be088F1c0",
```

Comment:
I would say the current implementation is https://testnet.bscscan.com/address/0x4baF88AbA7f86f6540728BEF454b224C5d2215e7